command/polybftmanifest/manifest_init.go

```diff
@@ -75,18 +74,11 @@ func setFlags(cmd *cobra.Command) {
 		"folder prefix names for polybft validator keys",
 	)
 
-	cmd.Flags().Int64Var(
-		&params.chainID,
-		chainIDFlag,
-		command.DefaultChainID,
-		"the ID of the chain",
-	)
```

Comment:
This is the problem why e2e test is failing:
we are providing different chain id when initializing CheckpointManager smart contract (https://github.com/0xPolygon/core-contracts/blob/dev/contracts/root/CheckpointManager.sol#L42) and when calculating checkpoint hash (which takes chain id as well into account: https://github.com/0xPolygon/polygon-edge/blob/develop/consensus/polybft/blockchain_wrapper.go#L183). Later one is read from Genesis configuration and it defaults to 100, whereas this one in manifest file is left to be 0 (since it is not populated and it is defaulted to 0). Therefore we have mismatch between checkpoint hash calculated on the smart contract vs the one calculated and signed by the client.